README.md

```diff
@@ -562,12 +584,15 @@ let username = "your-username-here"
 let password = "your-password-here"
 let speechToText = SpeechToText(username: username, password: password)
 
+var accumulator = SpeechRecognitionResultsAccumulator()
+
 func startStreaming() {
-    var settings = RecognitionSettings(contentType: .oggOpus)
+    var settings = RecognitionSettings(contentType: "audio/ogg;codecs=opus")
```

Comment:
Yeah, not a bad idea.

One thing to note is that some content-types also allow user-specifiable options. For example, `audio/l16;rate=16000;channels=2`. We can express that in Swift using an `enum` with associated values. But I'm not sure it can be expressed as an `enum` in Java.